solution/backend/resources/tests/test_admin.py

```diff
@@ -97,7 +97,8 @@ def test_add_supplemental_content(self):
             results = self.supplementalAdmin.add_content(output)
         good_section = Section.objects.get(section_id=200)
         good_category = AbstractCategory.objects.get(name="test")
-        row1 = {"name": "<a href=/admin/resources/supplementalcontent/1/change/ target='blank'>content-1</a>",
+        id = str(SupplementalContent.objects.get(name='content-1').id)
+        row1 = {"name": f"<a href=/admin/resources/supplementalcontent/{id}/change/ target='blank'>content-1</a>",
```

Comment:
this test was a little faulty.  If there was any supplemental content ever on the test db it fails since it wont be "1".  We just get the ID now before the check.